Cargo.toml

```diff
@@ -55,7 +55,7 @@ xz2 = "0.1.3"
 pgp = { version = "0.4", default-features = false}
 
 [dependencies.retry]
-version = "0.5"
+version = "1"
 default-features = false
```

Comment:
btw, it there the reason to separate `retry` section?

Reply:
Only because of aesthetics of the author who picked `retry` to use.